src/blah/it/address.gleam

```diff
@@ -0,0 +1,39 @@
+import gleam/string
+
+import blah/locales/it/address
+import blah/utils.{get_random_item}
+
+pub fn country() {
+  get_random_item(address.countries)
+}
+
+pub fn region() {
+  get_random_item(address.regions)
+}
+
+pub fn city() {
+  get_random_item(address.cities)
+}
+
+pub fn province() {
+  get_random_item(address.provinces)
+}
+
+pub fn street() {
+  let street = get_random_item(address.streets)
+  // most of Italian street names starts with "rue"        
```

Comment:
I don't understand. The prefix you're attaching is `Via`, but you say `rue` in the comment!

Reply:
I'm really sorry for the late response. 
You're totally right, it was my mistake